client/packages/programs/src/api/operations.graphql

```diff
@@ -269,9 +274,6 @@ query encounterById($storeId: String!, $encounterId: String!) {
       __typename
       nodes {
         ...Encounter
-        programEnrolment {
-          isImmunisationProgram
-        }
```

Comment:
Don't need these explicitly here, as they're in the "Encounter" fragment definition (above)